src/homework02/Computer.java

```diff
@@ -0,0 +1,41 @@
+package homework02;
+
+public class Computer {
+
+    private String HDD;
+    private String RAM;
+
+    public Computer(String HDD, String RAM) {
+        this.HDD = HDD;
+        this.RAM = RAM;
+    }
+
+    public String getRAM() {
+        return RAM;
+    }
+
+    public String getHDD() {
+        return HDD;
+    }
+
+    private Computer(ComputerBuilder builder) {
```

Comment:
Class should be created by builder not to use its own constructor to get builder. It is not his responsibility